### PR TITLE
Display a warning when using an invalid node version. r=vporof

### PR DESCRIPTION
**Pull request by jsantell**
Displays a message when users have an outdated version of node.

### CODE REVIEW

**Comment by jsantell** [comment]
```
*****
You are currently running node v5.0.0. The minimum supported version is v5.8.0. Please upgrade your version of node, or use a node version manager, like nvm:
https://github.com/creationix/nvm
*****
```

File: build/main.js [hunk 1]
```diff
@@ -14,8 +14,9 @@
 
 require('babel-polyfill');
 require('babel-register')();
-
+const semver = require('semver');
```
Comment (victorporof) [inline build/main.js]: nit: I'd like a newline here, but this is a personal preference so whatever you want.

**Comment by jsantell** [comment]
Updated to use package.json's `engines.node` property and displays the valid ranges.

**Comment by victorporof** [comment]
r+ hot hot!
appveyor failed for some reason, but ship away if all's good.

**Pull request merged**